content/en/docs/reference/command-line-tools-reference/feature-gates/kube-proxy-draining-terminating-nodes.md

```diff
@@ -13,6 +13,10 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.30"
+    toVersion: "1.31"
```

Comment:
The `toVersion` need correction to reflect the last version for the previous stage.

```suggestion
    toVersion: "1.30"
```

Reply:
ACK, I've applied the change

Reply:
I missed it, thanks @dipesh-rawat for the suggestion.